### PR TITLE
fix: change the over time to Over Time in README

### PR DESCRIPTION
**Pull request by juxianzheng**
Modify yu ju for contributor-graph repository

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,15 +4,15 @@ This tool is used to generate contributor graphs across repositories. We use Git
 
 ## Demo
 
-### Contributor over time for single repo
+### Contributor over Time for single repo
```
Comment (guoqqqi) [inline README.md:7]:
```suggestion
### Contributor Over Time for single repo
```

File: README.md [hunk 2]
```diff
@@ -4,15 +4,15 @@ This tool is used to generate contributor graphs across repositories. We use Git
...
 
-[![Contributor over time](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=apache/apisix)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=apache/apisix)
+[![Contributor over Time](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=apache/apisix)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=apache/apisix)
```
Comment (guoqqqi) [inline README.md:9]: ditto

File: README.md [hunk 3]
```diff
@@ -4,15 +4,15 @@ This tool is used to generate contributor graphs across repositories. We use Git
...
 
-### Contributor over time across related repo
+### Contributor over Time across related repo
```
Comment (guoqqqi) [inline README.md:11]: ditto

File: README.md [hunk 4]
```diff
@@ -4,15 +4,15 @@ This tool is used to generate contributor graphs across repositories. We use Git
...
 
 We maintain a [list](api-go/config/multi-repo.yaml) which you could directly add your repo list and commit a PR if needed.
 
-[![Contributor over time](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=apache/apisix&merge=true)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=apache/apisix&merge=true)
+[![Contributor over Time](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=apache/apisix&merge=true)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=apache/apisix&merge=true)
```
Comment (guoqqqi) [inline README.md:15]: ditto

**Comment by guoqqqi** [comment]
image
Note that there are spaces

**Comment by guoqqqi** [comment]
image
and also update here

File: README.md [hunk 5]
```diff
@@ -4,15 +4,15 @@ This tool is used to generate contributor graphs across repositories. We use Git
...
 ## Demo
 
-### Contributor over time for single repo
+### Contributor Over Time for single repo
 
-[![Contributor over time](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=apache/apisix)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=apache/apisix)
+[![Contributor Over Time](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=apache/apisix)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=apache/apisix)
 
-### Contributor over time across related repo
+### ContributorOver Time across related repo
```
Comment (juzhiyuan) [inline README.md:11]:
```suggestion
### Contributor Over Time across related repo
```

File: README.md [hunk 6]
```diff
@@ -8,7 +8,7 @@ This tool is used to generate contributor graphs across repositories. We use Git
 
 [![Contributor Over Time](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=apache/apisix)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=apache/apisix)
 
-### ContributorOver Time across related repo
+### Contributor Over Time  across related repo
```
Comment (guoqqqi) [inline README.md:11]:
```suggestion
### Contributor Over Time across related repo
```

File: README.md [hunk 7]
```diff
@@ -4,15 +4,15 @@ This tool is used to generate contributor graphs across repositories. We use Git
 
 ## Demo
 
-### Contributor over time for single repo
+### Contributor Over Time for single repo
 
-[![Contributor over time](https://contributor-overtime-api.git-contributor.com/contributors-svg?chart=contributorOverTime&repo=apache/apisix)](https://git-contributor.com?chart=contributorOverTime&repo=apache/apisix)
```
[the remark this answers is not in the document]
Comment (juzhiyuan) [reply README.md:9]: I would suggest @guoqqqi and @juxianzheng have clear expectations on this issue.

[the remark this answers is not in the document]
Comment (guoqqqi) [reply README.md:9]: @juxianzheng  Please check the file modified by this PR, it should only contain changes to `Over Time`, please restore other changes

File: README.md [hunk 8]
```diff
@@ -4,15 +4,15 @@ This tool is used to generate contributor graphs across repositories. We use Git
...
 
-### Contributor over time across related repo
+
+### Contributor Over Time across related repo
 
 We maintain a [list](api-go/config/multi-repo.yaml) which you could directly add your repo list and commit a PR if needed.
 
-[![Contributor over time](https://contributor-overtime-api.git-contributor.com/contributors-svg?chart=contributorOverTime&repo=apache/apisix&merge=true)](https://git-contributor.com?chart=contributorOverTime&repo=apache/apisix&merge=true)
+
```
Comment (guoqqqi) [inline README.md:15]:
```suggestion
[![Contributor over time](https://contributor-overtime-api.git-contributor.com/contributors-svg?chart=contributorOverTime&repo=apache/apisix&merge=true)](https://git-contributor.com?chart=contributorOverTime&repo=apache/apisix&merge=true)
```

**Review by guoqqqi — APPROVED** [review]

**Pull request merged**